### DISCUSSION TITLE
How do I render gui windows without the ugly windows host window

### DISCUSSION THREAD

**Discussion by refrai-n** [ocornut/imgui#6029] [New Users Build/Link/Run issues ONLY!]
This is probably a noobie question, I'm using the win32directx11 example from the code, when I build and run it, the 2 imgui windows are displayed in a generic parent window, I want to make my gui window without the ugly background and border and the default windows close minimize buttons etc. I just want a single independant gui window.
![sssss](https://user-images.githubusercontent.com/111191103/210001720-ed778c95-83fa-410b-8b83-0d8c5a99ff04.PNG)

**Comment by ocornut** [comment]
Sorry but #1586

**Comment by uzername** [comment]
Well, you'd like to look at borderless style of window. DirectX example features WinAPI code for creating a window (it's how DX11 works), and you may need to complement it with setting a window style: https://social.msdn.microsoft.com/Forums/vstudio/en-US/b98c4c06-9581-44d3-8e5a-4adb2316e653/win32-about-styles-how-can-i-do-a-borderless-window?forum=vclanguage . 
> To make a window borderless, you should not use WS_BORDER style. Setting NULL as window style means you as using WS_OVERLAPPED window style.An overlapped window has a title bar and a border. 

I think, it is set in CreateWindowEx call.

Removing a background from window is more challenging though.

**Reply by uzername** [reply]
I have found something regarding making transparent window: https://learn.microsoft.com/en-us/answers/questions/639888/how-to-make-transparent-window-using-updatelayered.html . That code is too comlicated
